lms/templates/courseware/accordion.html

```diff
@@ -40,9 +40,12 @@
                 ## a status icon
                 <%
                     if section.get('due') is None:
-                        data_string = ''
+                        data_string = _(section['format'])
                     else:
-                        data_string = _('due {date}')
+                        if 'proctoring' in section:
+                            data_string = _('due {date}')
+                        else:
+                            data_string = _(section['format'] + " due {date}")
```

Comment:
Again, this translation is problematic. The correct thing would be to use a placeholder for section['format']). Something like _("{section_format} due {date}".format(section_format=section['format'], date=...).